backend/app/api/v1/endpoints/telco/telcoJobs.py

```diff
@@ -60,8 +67,8 @@ async def jobs(
         "startDate": start_date.__str__(),
         "endDate": end_date.__str__(),
         "results": jobs,
-        "total": 0,
-        "offset": 0,
+        "total": results["total"],
+        "offset": offset + size if size else 0,
```

Comment:
Is it intentional that this differs from the `ocpJobs` logic above, where we're checking for `size != 10000` rather than just `if size`? If `offset` is 0, and `size` isn't specified, that one will return 0 and this will return 10000.

Reply:
It slipped my mind to add the `size != 10000` here. Thanks for catching it!